src/nvim/ex_getln.c

```diff
@@ -987,7 +987,8 @@ static uint8_t *command_line_enter(int firstc, long count, int indent, bool init
 
   if (!tl_ret && ERROR_SET(&err)) {
     msg_putchar('\n');
-    msg_printf_attr(HL_ATTR(HLF_E)|MSG_HIST, (char *)e_autocmd_err, err.msg);
+    emsg(e_autocmd_err);
+    msg(err.msg);
```

Comment:
maybe `emsg` so the rest also printed with `ErrorMsg` color

Reply:
`e_autocmd_err` has a `%s` in it, so it expects an argument. How about `semsg` ?:

    semsg(e_autocmd_err, err.msg);